cgrok/grok.go

```diff
@@ -1,8 +1,9 @@
 package grok
 
 /*
-#cgo LDFLAGS: -lgrok
-#include <grok.h>
+#cgo CFLAGS: -I. -std=gnu99
+#cgo windows LDFLAGS: -L. -lportablexdr -lws2_32
+#include "grok.h"
```

Comment:
`-I.` should look in the local directory for headers. According to https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html GCC will look at the -I _before_ the system paths, so we've got that covered.